### PR TITLE
Add XDomainRequest detection

### PR DESCRIPTION
**Pull request by hypotenuse**

### CODE REVIEW

File: feature-detects/xdomainrequest.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    "name": "MSDN documentation",
+    "href": "https://msdn.microsoft.com/library/ie/cc288060.aspx/"
+  }]
+}
+!*/
+/* DOC
+Detects support for XDomainReuqest in ie9 & ie8
+*/
+define(['Modernizr'], function (Modernizr) {
+  Modernizr.addTest('xdomainrequest', 'XDomainRequest' in window && typeof XDomainRequest == 'function');
```
Comment (patrickkettner) [inline feature-detects/xdomainrequest.js]: is `typeof XDomainRequest` ever `!== 'function'`?

**Comment by patrickkettner** [comment]
hey @hypotenuse 

so sorry you were ignored for so long. one question, otherwise :+1:

File: feature-detects/xdomainrequest.js (hunk 1 above)
Comment (hypotenuse) [inline feature-detects/xdomainrequest.js]: No, I guess this may be omitted.

Comment (patrickkettner) [inline feature-detects/xdomainrequest.js]: would you be able to remove it?

**Comment by paulirish** [comment]
fixed that niggling thing: cd6e9a100df2d830df58da42eef626e4e3122dd8

**Comment by patrickkettner** [comment]
thanks, paul!

On Tue, Oct 20, 2015 at 4:00 PM, Paul Irish <notifications@github.com>
wrote:

> fixed that niggling thing: cd6e9a1
> <https://github.com/Modernizr/Modernizr/commit/cd6e9a100df2d830df58da42eef626e4e3122dd8>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Modernizr/Modernizr/pull/1566#issuecomment-149726220>.
>



-- 
patrick

**Pull request merged**